package.json

```diff
@@ -101,6 +105,8 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "svg-spritemap-webpack-plugin": "4.4.0",
+    "tablemark": "3.0.0",
```

Comment:
I also didn't find these 2 dependencies in the code:

`tablemark` and `turbo-combine-reducers`

Reply:
I am going to remove `tablemark`,  "markdown-table",  "markdown-table-formatter".

I was trying to make my script generate a markdown but gave up it due  lack of time.

Reply:
turbo-combine-reducers is used by `@wordpress/data`. I was having some issues with it but now I see that is possible to remove, so removed!

Reply:
Remove here:
https://github.com/Automattic/sensei/pull/5159/commits/6e95f28539c96167f4e7a18b75958bdefff47e29